src/Chip/Chip.test.jsx

```diff
@@ -77,5 +77,37 @@ describe('<Chip />', () => {
       await userEvent.type(iconAfter, '{enter}');
       expect(func).toHaveBeenCalled();
     });
+    it('onIconBeforeClick is triggered', async () => {
+      const func = jest.fn();
+      render(
+        <TestChip iconBefore={Close} onIconBeforeClick={func} />,
+      );
+      const iconBefore = screen.getByTestId('icon-before');
+      await userEvent.click(iconBefore);
+      expect(func).toHaveBeenCalled();
```

Comment:
Can we avoid using test ids here? You can find these buttons by alt text for example, which should be a required prop now that's missing in your tests 🙂